metadata-ingestion/tests/unit/test_redash_source.py

```diff
@@ -685,8 +685,8 @@ def test_get_chart_snapshot_parse_table_names_from_sql(mocked_data_source):
                 ),
                 chartUrl="http://localhost:5000/queries/4#10",
                 inputs=[
-                    "urn:li:dataset:(urn:li:dataPlatform:mysql,Rfam.orders,PROD)",
                     "urn:li:dataset:(urn:li:dataPlatform:mysql,Rfam.order_items,PROD)",
+                    "urn:li:dataset:(urn:li:dataPlatform:mysql,Rfam.orders,PROD)",
```

Comment:
Why is this necessary? Would this sort order requirement break any of the existing customers doing custom transforms?